modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About the Loki Operator
+[role="_abstract"]
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+= Deploying the Lokistack
+You can use the {product-title} web console to install the LokiOperator.
```

Comment:
These two lines are appearing on the same line on the preview docs.

Reply:
LokiOperator seems strange.  For the first instance, at least, I'd expect something like this: 
```suggestion
You can use the {product-title} web console to install the Loki Operator named "LokiOperator."
```

Reply:
Resolved. looks good to now